tests/test_harvest_bookmarks.py

```diff
@@ -645,16 +646,16 @@ def do_test(self, conn_id):
                                 target_value = self.local_to_utc(dt.utcfromtimestamp(target_value))
 
                     except (OverflowError, ValueError, TypeError):
-                        print("bookmarks cannot be converted to dates, comparing values directly")
+                        print("Bookmarks cannot be converted to dates, comparing values directly")
```

Comment:
Please change this with tap-tester LOGGER

Reply:
Replaced `print` with LOGGER.